services/analytics/profile.go

```diff
@@ -228,7 +211,6 @@ func makeStats(metricNameRoot string, total, res models.M, numQueries float32, p
 		Rate:      rate,
 		Cnt:       interfaceToFloat32(res["m_"+metricNameRoot+"_cnt"]),
 		Sum:       sum,
-		Avg:       sum / numQueries,
```

Comment:
looks like we still should keep this for backward compatibility

Reply:
Added back, but in the query for better performance.